src/tests/unittests/configuration_helper/adapters/test_d5a_instrument_adapter.py

```diff
@@ -61,20 +63,43 @@ def test_apply_config(self):
             self.assertEqual(d5a_module_mock(), d5a_adapter.instrument.d5a)
             self.assertEqual(instrument_name, d5a_adapter.instrument.name)
 
+            mocked_snapshot = {'name': 'd5a', 'parameters': self.mock_config}
+            d5a_adapter.instrument.snapshot = MagicMock(return_value=mocked_snapshot)
```

Comment:
Why does the snapshot need to be mocked now?

Reply:
read config returns snapshot.